x/gov/testutil/expected_keepers.go

```diff
@@ -26,6 +26,13 @@ type BankKeeper interface {
 	bankkeeper.Keeper
 }
 
+// PoolKeeper extends the gov's actual expected PoolKeeper.
+type PoolKeeper interface {
+	types.PoolKeeper
+
+	FundCommunityPool(ctx context.Context, amount sdk.Coins, sender sdk.AccAddress) error
```

Comment:
This method is already present in `types.PoolKeeper` why to specify again?

Reply:
removed